sdk/core/azure-core/inc/uuid.hpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <new> // for placement new
+#include <string>
+#include <utility> // for swap and move
+#include <random>
```

Comment:
Why we still need `<new>` and `<utility>` here? We didn't use placement new and swap in this header.